src/manifest/manifest_test.ts

```diff
@@ -1172,3 +1173,66 @@ Deno.test("Manifest throws error when CustomEvents with duplicate name are added
     assertStringIncludes(error.message, "CustomEvent");
   }
 });
+
+Deno.test("Manifest correctly parses valid domains", () => {
+  const def = {
+    outgoingDomains: [
+      "https://slack.com",
+      "http://salesforce.com",
+      "https://api.slack.com",
```

Comment:
I'd add some more test cases here:

1. Bare IP addresses
2. Fully qualified domain names, including subdomains, i.e. en.wikipedia.org
3. Either of the above w/ port specifiers
4. All of the above with and without protocol specifiers